sigmahq/sigmahq_conventions.md

```diff
@@ -93,3 +96,20 @@ detection:
             - '\example_2.exe'
             - '\example_3.exe'
 ```
+
+## False Postivess
+
+- If the rule author expects false positives, then its must be expressed. For example:
+
+```yml
+falsepositives:
+    - During software installation
+    - Legitimate usage of the tool
```

Comment:
```suggestion
    - Legitimate administrative activity
```
I think "legitimate usage of the tool" is a low-value false positive text as it is almost always true (except for stuff like mimikatz execution or whatever)